test/test_other.py

```diff
@@ -13317,6 +13317,22 @@ def test_emscripten_set_immediate(self):
   def test_emscripten_set_immediate_loop(self):
     self.do_runf('emscripten_set_immediate_loop.c')
 
+  @parameterized({
+    '': ([],),
+    'pthreads': (['-pthread', '-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME'],),
+  })
+  def test_emscripten_main_loop(self, args):
+    self.do_runf('test_emscripten_main_loop.c', emcc_args=args)
+
+  def test_emscripten_main_loop_and_blocker(self):
+    self.do_runf('test_emscripten_main_loop_and_blocker.c')
+
+  def test_emscripten_main_loop_settimeout(self):
+    self.do_runf('test_emscripten_main_loop_settimeout.c')
+
+  def test_emscripten_main_loop_setimmediate(self):
+    self.do_runf('test_emscripten_main_loop_setimmediate.c')
```

Comment:
Maybe we don't need them all - do you have a sense that these are important to test in node?

Reply:
Personally I would like to have all tests that can run under node, run under node.   I don't think these will have much time, especially since they are in other rather than code.  Browser-only tests are much hard for me (and I would assume others) to debug.